test/preprocessing/test_preprocessing.py

```diff
@@ -42,6 +45,51 @@ def test_verify_pixel_variation(self):
         self.assertGreater(np.amax(self.raw), np.amin(self.raw))
         self.assertGreater(np.amax(self.comp), np.amin(self.comp))
 
+    def test_truncate_filenames(self):
+        """
+        Test that truncate_filenames does not crash
+
+        This test creates a directory structure with an image in it,
+        truncates filenames inside it, and basically checks that it
+        does not crash.
+
+        Ideally, this test would create a file with too long filename,
+        run truncate_filenames, and check that the filename now is
+        sufficiently short. However, it was hard to create a file wih
+        long filename, so I did not spend more time on this.
```

Comment:
This explanation should not reside in the doctring, but rather as an issue (with the label enhancement)

Reply:
I think the `f"{'a'*255}.jpg"` solution will fix this though